docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_FloatingActions.htm

```diff
@@ -37,10 +37,11 @@ <h2><a id="manual-constructor"></a>Manual Constructor</h2>
    &lt;button class=&quot;ui-floatingactions-item&quot; data-icon=&quot;floating-add&quot;/&gt;
    &lt;button class=&quot;ui-floatingactions-item&quot; data-icon=&quot;floating-search&quot;/&gt;
 &lt;/div&gt;
-&lt;script&gt;
-   var elFloatingActions = document.getElementById(&quot;floating&quot;),
-       flaotingActions = tau.widget.FloatingActions(elFloatingActions);
-&lt;/script&gt;
+</pre>
+<p>JS code:</p>
```

Comment:
Global Suggestion : How about adding "\<p>HTML code:\</p>" as "\<p>JS code:\</p>" added?

If you want to keep current state, I'll merge this PR.

Reply:
We have established the rules:
- for a single block of code we don't use description about code language
- if there are two blocks of code after each other then we use both descriptions HTML and JS

We will prepare fixes.